docker/devnet.conf

```diff
@@ -26,7 +26,7 @@ alephium.consensus.block-target-time = 10 millis
 alephium.consensus.uncle-dependency-gap-time = 0 seconds
 alephium.network.leman-hard-fork-timestamp = 1643500800000 // GMT: 30 January 2022 00:00:00
 
-alephium.network.network-id = 5
+alephium.network.network-id = 4
```

Comment:
Will address the config issue in a seperate PR
This requires an update to full node devnet config template, which affected a lot of tests.